app/models/order.rb

```diff
@@ -4,6 +4,10 @@ class Order < ApplicationRecord
   include Api::V1x0::Catalog::DiscardRestore
   destroy_dependencies :order_items
   acts_as_tenant(:tenant)
+  attribute :state, :string, :default => 'Created'
+  validates_inclusion_of :state,
+    :in => ["Created", "Approval Pending", "Ordered", "Failed", "Completed", "Canceled"].freeze,
```

Comment:
Too bad "Approval Pending" breaks the use of `%w` here.  😞 

I sense that we should at least order them alphabetically.